api/controllers/linux/install/infra.go

```diff
@@ -67,6 +67,10 @@ func (c *InstallController) SetupInfra(ctx context.Context, ignoreHostPreflights
 			}
 		}()
 
+		if err := c.updateInfraManagerWithLatestConfigValues(); err != nil {
```

Comment:
why not simply get the values and pass them to the `Install(` function below?

Reply:
felt like that was going to sprawl into a lot more changes if the function signature needed to change. can do that thought if its what we want